oioswift/proxy/controllers/container.py

```diff
@@ -229,14 +238,23 @@ def get_container_head_resp(self, req):
 
         return resp
 
-    def load_container_metadata(self, headers):
-        metadata = {}
-        metadata.update(
-            (k, v)
-            for k, v in headers.iteritems()
+    def properties_from_headers(self, headers):
+        metadata = {
+            k: v
+            for k, v in headers.items()
             if k.lower() in self.pass_through_headers or
-            is_sys_or_user_meta('container', k))
-        return metadata
+            is_sys_or_user_meta('container', k)
+        }
+
+        system = dict()
+        # This header enables versioning
+        ver_loc = headers.get('X-Container-Sysmeta-Versions-Location')
```

Comment:
Do we receive header without value (i.e. "") or header is discarded previously ? (and ver_loc is None)
because
```
>>> print("" is not None)
True
```